CHANGELOG.md

```diff
@@ -276,6 +283,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 [Issue #1324]: https://bugs.bareos.org/view.php?id=1324
 [Issue #1329]: https://bugs.bareos.org/view.php?id=1329
 [Issue #1369]: https://bugs.bareos.org/view.php?id=1369
+[Issue #1374]: https://bugs.bareos.org/view.php?id=1374
```

Comment:
These links will automatically be updated on release. Please do not update these links as they create more conflicts in CHANGELOG.md.